#! /bin/sh -e

## DP: Small changes to dovecot-example.conf
## DP: Author: Jaldhar H. Vyas <jaldhar@debian.org>

. $(dirname $0)/DPATCH

exit 0
@DPATCH@
diff -urN trunk/dovecot-example.conf trunk.debian/dovecot-example.conf
--- trunk/dovecot-example.conf	2007-03-25 13:58:04.000000000 +0200
+++ trunk.debian/dovecot-example.conf	2007-03-25 14:08:36.000000000 +0200
@@ -18,6 +18,7 @@
 # Protocols we want to be serving: imap imaps pop3 pop3s
 # If you only want to use dovecot-auth, you can set this to "none".
 #protocols = imap imaps
+protocols =
 
 # IP or host address where to listen in for connections. It's not currently
 # possible to specify multiple addresses. "*" listens in all IPv4 interfaces.
@@ -67,6 +68,7 @@
 # Prefix for each line written to log file. % codes are in strftime(3)
 # format.
 #log_timestamp = "%b %d %H:%M:%S "
+log_timestamp = "%Y-%m-%d %H:%M:%S "
 
 # Syslog facility to use if you're logging to syslog. Usually if you don't
 # want to use "mail", you'll use local0..local7. Also other standard
@@ -86,8 +88,7 @@
 
 # PEM encoded X.509 SSL/TLS certificate and private key. They're opened before
 # dropping root privileges, so keep the key file unreadable by anyone but
-# root. Included doc/mkcert.sh can be used to easily generate self-signed
-# certificate, just make sure to update the domains in dovecot-openssl.cnf
+# root.
 #ssl_cert_file = /etc/ssl/certs/dovecot.pem
 #ssl_key_file = /etc/ssl/private/dovecot.pem
 
@@ -200,7 +201,7 @@
 #   %d - domain part in user@domain, empty if there's no domain
 #   %h - home directory
 #
-# See doc/variables.txt for full list. Some examples:
+# See /usr/share/doc/dovecot-common/variables.txt for full list. Some examples:
 #
 #   mail_location = maildir:~/Maildir
 #   mail_location = mbox:~/mail:INBOX=/var/mail/%u
@@ -251,7 +252,10 @@
 
 # Grant access to these extra groups for mail processes. Typical use would be
 # to give "mail" group write access to /var/mail to be able to create dotlocks.
+# The dovecot processes get these groups' privileges in addition to the dovecot
+# ones; the list of groups must be separated by spaces.
 #mail_extra_groups =
+mail_extra_groups = mail
 
 # Allow full filesystem access to clients. There's no access checks other than
 # what the operating system does for the active UID/GID. It works with both
@@ -267,7 +271,8 @@
 # isn't finding your mails.
 #mail_debug = no
 
-# Log prefix for mail processes. See doc/variables.txt for list of possible
+# Log prefix for mail processes. See
+# /usr/share/doc/dovecot-common/variables.txt for list of possible
 # variables you can use.
 #mail_log_prefix = "%Us(%u): "
 
@@ -497,19 +502,19 @@
 
 protocol imap {
   # Login executable location.
-  #login_executable = /usr/libexec/dovecot/imap-login
+  #login_executable = /usr/lib/dovecot/imap-login
 
   # IMAP executable location. Changing this allows you to execute other
   # binaries before the imap process is executed.
   #
   # This would write rawlogs into ~/dovecot.rawlog/ directory:
-  #   mail_executable = /usr/libexec/dovecot/rawlog /usr/libexec/dovecot/imap
+  #   mail_executable = /usr/lib/dovecot/rawlog /usr/lib/dovecot/imap
   #
   # This would attach gdb into the imap process and write backtraces into
   # /tmp/gdbhelper.* files:
   #   mail_executable = /usr/libexec/dovecot/gdbhelper /usr/libexec/dovecot/imap
   #
-  #mail_executable = /usr/libexec/dovecot/imap
+  #mail_executable = /usr/lib/dovecot/imap
 
   # Maximum IMAP command line length in bytes. Some clients generate very long
   # command lines with huge mailboxes, so you may need to raise this if you get
@@ -519,7 +524,7 @@
   # Support for dynamically loadable plugins. mail_plugins is a space separated
   # list of plugins to load.
   #mail_plugins = 
-  #mail_plugin_dir = /usr/lib/dovecot/imap
+  #mail_plugin_dir = /usr/lib/dovecot/modules/imap
 
   # Send IMAP capabilities in greeting message. This makes it unnecessary for
   # clients to request it with CAPABILITY command, so it saves one round-trip.
@@ -562,11 +567,11 @@
 
 protocol pop3 {
   # Login executable location.
-  #login_executable = /usr/libexec/dovecot/pop3-login
+  #login_executable = /usr/lib/dovecot/pop3-login
 
   # POP3 executable location. See IMAP's mail_executable above for examples
   # how this could be changed.
-  #mail_executable = /usr/libexec/dovecot/pop3
+  #mail_executable = /usr/lib/dovecot/pop3
 
   # Don't try to set mails non-recent or seen with POP3 sessions. This is
   # mostly intended to reduce disk I/O. With maildir it doesn't move files
@@ -611,7 +616,7 @@
   # installations. %08Xu%08Xv will be the new default, so use it for new
   # installations.
   #
-  #pop3_uidl_format = 
+  pop3_uidl_format = %08Xu%08Xv
 
   # POP3 logout format string:
   #  %t - number of TOP commands
@@ -626,7 +631,7 @@
   # Support for dynamically loadable plugins. mail_plugins is a space separated
   # list of plugins to load.
   #mail_plugins = 
-  #mail_plugin_dir = /usr/lib/dovecot/pop3
+  #mail_plugin_dir = /usr/lib/dovecot/modules/pop3
 
   # Workarounds for various client bugs:
   #   outlook-no-nuls:
@@ -643,9 +648,9 @@
 ## LDA specific settings
 ##
 
-protocol lda {
+# protocol lda {
   # Address to use when sending rejection mails.
-  postmaster_address = postmaster@example.com
+  # postmaster_address = postmaster@example.com
 
   # Hostname to use in various parts of sent mails, eg. in Message-Id.
   # Default is the system's real hostname.
@@ -654,21 +659,24 @@
   # Support for dynamically loadable plugins. mail_plugins is a space separated
   # list of plugins to load.
   #mail_plugins = 
-  #mail_plugin_dir = /usr/lib/dovecot/lda
+  #mail_plugin_dir = /usr/lib/dovecot/modules/lda
 
   # Binary to use for sending mails.
   #sendmail_path = /usr/lib/sendmail
 
   # UNIX socket path to master authentication server to find users.
   #auth_socket_path = /var/run/dovecot/auth-master
-}
+
+  # Enabling Sieve plugin for server-side mail filtering
+  # mail_plugins = cmusieve
+# }
 
 ##
 ## Authentication processes
 ##
 
 # Executable location
-#auth_executable = /usr/libexec/dovecot/dovecot-auth
+#auth_executable = /usr/lib/dovecot/dovecot-auth
 
 # Set max. process size in megabytes.
 #auth_process_size = 256
@@ -802,7 +810,8 @@
     # because PAM modules can do all kinds of checks besides checking password,
     # such as checking IP address. Dovecot can't know about these checks
     # without some help. cache_key is simply a list of variables (see
-    # doc/variables.txt) which must match for the cached data to be used.
+    # /usr/share/doc/dovecot-common/variables.txt) which must match for
+    # the cached data to be used.
     # Here are some examples:
     #   %u - Username must match. Probably sufficient for most uses.
     #   %u%r - Username and remote IP address must match.
@@ -858,15 +867,15 @@
   # SQL database
   # http://wiki.dovecot.org/AuthDatabase/SQL
   #passdb sql {
-    # Path for SQL configuration file, see doc/dovecot-sql-example.conf
-    #args = 
+    # Path for SQL configuration file
+    #args = /etc/dovecot/dovecot-sql.conf
   #}
 
   # LDAP database
   # http://wiki.dovecot.org/AuthDatabase/LDAP
   #passdb ldap {
-    # Path for LDAP configuration file, see doc/dovecot-ldap-example.conf
-    #args = 
+    # Path for LDAP configuration file
+    #args = /etc/dovecot/dovecot-ldap.conf
   #}
 
   # vpopmail authentication
@@ -923,15 +932,15 @@
   # SQL database
   # http://wiki.dovecot.org/AuthDatabase/SQL
   #userdb sql {
-    # Path for SQL configuration file, see doc/dovecot-sql-example.conf
-    #args = 
+    # Path for SQL configuration file
+    #args = /etc/dovecot/dovecot-sql.conf
   #}
 
   # LDAP database
   # http://wiki.dovecot.org/AuthDatabase/LDAP
   #userdb ldap {
-    # Path for LDAP configuration file, see doc/dovecot-ldap-example.conf
-    #args = 
+    # Path for LDAP configuration file
+    #args = /etc/dovecot/dovecot-ldap.conf
   #}
 
   # vpopmail
@@ -991,6 +1000,18 @@
       #mode = 0660
     #}
   #}
+
+  ## dovecot-lda specific settings
+  ##
+  # socket listen {
+  #   master {
+  #     path = /var/run/dovecot/auth-master
+  #     mode = 0600
+  #     user = mail # User running Dovecot LDA
+  #     #group = mail # Or alternatively mode 0660 + LDA user in this group
+  #   }
+  # }
+
 }
 
 # If you wish to use another authentication server than dovecot-auth, you can
